cartography/intel/gcp/compute.py

```diff
@@ -525,7 +548,7 @@ def _parse_port_string_to_rule(port: Optional[str], protocol: str, fw_partial_ur
     }
 
 
-@timeit
+@ timeit
```

Comment:
there should not be space between @ and timeit. can you fix everywhere?

Reply:
changes are done